### ISSUE TITLE
Improving newcomer experience

### ISSUE THREAD

**Issue by kaustubh-nair** [publiclab/mapknitter#881]
I've noticed recently that Mapknitter lacks contribution from newcomers. Over the past few months that I've worked on this repo, I haven't noticed anyone new contributing actively even though plots2 has a ton of newcomers. I presumed it could be due to lack of awareness about the existence of mapknitter, but that is not true because all our FTOs are taken up by a lot of newcomers. 
I realized what mapknitter really lacks is _easy_ issues. I'm not talking about FTOs. I'm referring to simple and small issues that are easy to solve, and involve a little bit of coding so it helps people get familiar with the codebase. FTOs include step by step instructions so it's not really enough for someone who wants to start contributing to bigger issues.
@publiclab/reviewers Maybe we could be doing something about this so we can increase activity in mapknitter! :tada: :smile:

**Comment by gauravano** [comment]
Sure @kaustubh-nair, we all can try to make issues as easy as possible by providing detailed solution. If you have any other Outreach strategy in mind, feel free to mention.

**Comment by jywarren** [comment]
Agreed, this would be really nice! @divyabaid16 has been opening a variety of issues. Maybe some of those can be converted into first-timers-only issues?

Thank you for thinking of this!

**Comment by divyabaid16** [comment]
And yes, maybe we can also create alternate issues, like first, we may create a FTO, with proper steps to include first-timers and then second time, we may just create a ```help-wanted``` issue, as simple as FTO but without the detailed steps to follow which would give a new contributor a chance to explore the codebase and solve it themselves.
Thanks!
